hardware/hardware.mk

```diff
@@ -89,8 +89,7 @@ include $(BSG_MANYCORE_DIR)/machines/arch_filelist.mk
 # but transformed into a tool-specific syntax where necesssary.
 VINCLUDES += $(HARDWARE_PATH)
 
-VHEADERS += $(BSG_MANYCORE_DIR)/testbenches/common/v/bsg_manycore_mem_cfg_pkg.v
```

Comment:
The nice thing about this change is that we no longer depend on any simulation-specific files in manycore. Just architectural files (arch_filelist.mk)